src/Components/AddPokemon/AddPokemon.js

```diff
@@ -1,37 +1,70 @@
-import React from "react";
+import React, { useState } from "react";
 import { useCreateCard } from "../../Helpers/apiCalls";
+import "./AddPokemon.css";
+import { validateCard } from "../../Helpers/formValidator";
 
 export default function AddPokemon(props) {
-    const { sets, types, updateCardData, cardData } = props;
+    const { sets, types } = props;
+
+    const newCardDataDefault = {
+        image: "",
+        pokedexIndex: "",
+        name: "",
+        type: 0,
+        HP: "",
+        pokemonCardSet: 0,
+    };
+
+    const [newCardData, setNewCardData] = useState(newCardDataDefault);
+
+    const [validationMessage, setValidationMessage] = useState("");
+
+    function updateCardData(event) {
```

Comment:
since this function is used in all 3 forms (the only difference being the setter function being used) maybe you can make it slightly more generic by making it take the event and the setter function and then using it  as a helper function or a fuunction passed in as a prop?